app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainViewModel.kt

```diff
@@ -114,11 +116,13 @@ internal class MainViewModel(
 
     private fun onParsingDone() {
         if (!repository.readScheduleChangesSeen()) {
-            val scheduleVersion = repository.readMeta().version
-            val sessions = repository.loadChangedSessions().toSessionsAppModel()
-            val statistic = ChangeStatistic.of(sessions, logging)
-            val parameter = ScheduleChangesParameter(scheduleVersion, statistic)
-            mutableScheduleChangesParameter.sendOneTimeEvent(parameter)
+            val changedSessions = repository.loadChangedSessions().toSessionsAppModel()
+            if (changedSessions.isNotEmpty()) {
```

Comment:
Why did you add this check? Is `repository.readScheduleChangesSeen()` returning `false` and `repository.loadChangedSessions()` returning an empty list a thing that can happen?

Reply:
This is an attempt to mitigate a scenario where the schedule changes dialog shows up **without actual changes**. See https://github.com/EventFahrplan/EventFahrplan/issues/599. To fix the base reason conceptual changes are needed. I am happy to talk about them outside of this PR.